cda.py

```diff
@@ -61,7 +65,10 @@ def main_deploy(args: argparse.Namespace) -> None:
 
 def main_snapshot(args: argparse.Namespace) -> None:
     args = parse_args()
-    cc = ClustersConfig(args.config, args.worker_range)
+    cc = ClustersConfig(
+        args.config,
+        worker_range=args.worker_range,
+    )
```

Comment:
```
cat pyproject.toml
[tool.black]
line-length = 250
```

No need to break lines until it's 250 long.

Reply:
this patch is already merged: https://github.com/bn222/cluster-deployment-automation/pull/242